docs/quilt/README.md

```diff
@@ -0,0 +1,67 @@
+
+File type | Filename Extension 
+----------|-----------------
+Microscope images | raw_converted.ome.zarr 
+All cells mask | all_cells_mask.ome.zarr 
+H2B nuclei segmentation mask | H2B_nuclear_segmentation.ome.zarr 
+EOMES nuclei segmentation mask | EOMES_nuclear_segmentation.ome.zarr 
+CytoGFP ground truth segmentation mask | cytoGFP_ground_truth_segmentation.ome.zarr 
+CollagenIV ground truth segmentation mask | collagenIV_ground_truth_segmentation.ome.zarr 
+CollagenIV segmentation probability | collagenIV_segmentation_probability.ome.zarr 
+CollagenIV segmentaion mesh | collagenIV_segmentation_mesh 
+
+For improved data accessibility we provide further information in the `manifests` section below.
+
+## manifests
+
+The `manifests` directory contains 6 manifests for streamlined data accessibility. Each manifest file and its contents are explained below. 
+
+* imaging_and_segmentation_data.csv: contains both the links to download and links to visualize without download, all the imaging data, segmentation data, and mesh data. This csv also provides columns that contain all accompanying metadata from this study.  
+
+* Imaging_and_segmentation_data_column_description.csv: A complete description of each column label present in the imaging_and_segmentation_data.csv is provided here. 
+
+* Image_analysis_extracted_features.csv: contains all the features extracted from the raw_converted.ome.zarr files using the corresponding all_cells_mask.ome.zarr segmentation files. Further, the metrics computed from the intensity trajectories over time for each movie is also provided. Features and metrics are given for each time point and Z-plane combination, respectively for each movie. 
+
+* Image_analysis_extracted_features_column_description.csv: A complete description of each column label present in the Image_analysis_extracted_features.csv is provided here. 
+
+* Migration_timing_through_mesh_extracted_features.csv: contains class annotations for the centroids of individual nuclei, indicating whether each centroid is inside or outside the CollagenIV_segmentation_mesh. 
+
+* Migration_timing_through_mesh_extracted_features_column_description.csv: A complete description of each column label present in the Migration_timing_through_mesh_extracted_features.csv is provided here. 
+ 
+
+## supplemental_files
+
+The `supplemental_files` directory contains model checkpoints (cytodl_checkpoints) and CytoDL configuration files (cytodl_configuration_files) used for all cells mask and collagenIV mask generation. 
```

Comment:
Isn't there a model for ZO1 segmentation too?

Reply:
No, we did not include ZO1 segmentation code or data in this release.